ecl2df/common.py

```diff
@@ -73,25 +78,29 @@
 # This is a magic filename that means read/write from/to stdout
 # This makes it impossible to write to a file called "-" on disk
 # but that would anyway create a lot of other problems in the shell.
-MAGIC_STDOUT = "-"
+MAGIC_STDOUT: str = "-"
 
-logger = logging.getLogger(__name__)
+logger: logging.Logger = logging.getLogger(__name__)
 
 
 def write_dframe_stdout_file(
-    dframe, output, index=False, caller_logger=None, logstr=None
-):
+    dframe: pd.DataFrame,
+    output: str,
+    index: bool = False,
+    caller_logger: logging.Logger = None,
```

Comment:
`Optional`?

Reply:
Yes. Not sure why `mypy` accepts this.

Reply:
I was wondering about the same - looks like an extra commandline value is needed:
https://mypy.readthedocs.io/en/stable/command_line.html#none-and-optional-handling

Maybe we should add that flag in `webviz-config`. Currently we are only using these flags over in that repository:
```
mypy --package webviz_config --ignore-missing-imports --disallow-untyped-defs --show-error-codes
```

Reply:
Alternatively maybe simply use the `mypy` default practice (infer `Optional` from `None` default). I did not know it was a "feature" in `mypy`.